pkg/operator/minio/controller.go

```diff
@@ -207,14 +242,13 @@ func validateObjectStoreSpec(spec miniov1alpha1.ObjectStoreSpec) error {
 	return nil
 }
 
-func (c *MinioController) makeMinioStatefulSet(name, namespace string, spec miniov1alpha1.ObjectStoreSpec, ownerRef meta_v1.OwnerReference) (*v1beta2.StatefulSet, error) {
-	appsClient := c.context.Clientset.AppsV1beta2()
-
+func (c *Controller) makeMinioStatefulSet(name, namespace string, spec miniov1alpha1.ObjectStoreSpec, ownerRef meta_v1.OwnerReference) (*v1beta2.StatefulSet, error) {
 	accessKey, secretKey, err := c.getAccessCredentials(spec.Credentials.Name, spec.Credentials.Namespace)
 	if err != nil {
 		return nil, err
 	}
 
+	// TODO Use a Secret object (`envFrom`) instead of putting the keys in plain text in the StatefulSet
```

Comment:
instead of a TODO, is there an issue tracking this work item?

Reply:
Created: https://github.com/rook/rook/issues/2465